internal/clients/http/gcp/gcp_client.go

```diff
@@ -307,3 +309,9 @@ func (c *gcpClient) GetInstanceDescriptionByID(ctx context.Context, id, zone str
 	}
 	return &instanceDesc, nil
 }
+
+func getPubkeyWithUsername(ctx context.Context, pubkey string) string {
+	pubkey = strings.TrimSpace(pubkey)
+	parts := strings.Split(pubkey, " ")
+	return fmt.Sprintf("%s:%s %s", "gcp-user", parts[0], parts[1])
```

Comment:
Interesting :-D Why not:

```suggestion
	return fmt.Sprintf("gcp-user:%s %s", parts[0], parts[1])
```